### PR TITLE
Project Clean Up

### PR DESCRIPTION
**Pull request by irish1986**
## Pull Request Checklist

- [x] I have run `yarn build` to ensure that the project builds successfully.
- [ ] I have updated `CHANGELOG.md` with the necessary changes made in this pull request.
  - Unsure how to treat "architecture" change in respect with the `CHANGELOG.md`, please advice.

## Description

* Create & Move **GitHub** related files
  * GitHub related files are better located undre `.github` directory
  * Mostly for future community guidance and participation (placeholder for now)
* `Contribution Guideline` under `.github` directory
  * Renamed `README_DEV.md` → `.github/CONTRIBUTING.md`
* `PR Template` under `.github` directory
  * Delete `pull-request-template.md` duplicate
  * Rename `pull_request_template.md` to `.github/PULL_REQUEST_TEMPLATE.md`]

## Related Issues

I create discusssion #225 regarding this awesome project.  Wanted to give back a bit by offering some of my know how.... Given I don't know a thing about sewing.

If welcomed, I would like to implement more DevOps capabilities around CI pipeline, pre-commit-hooks, dependencies management, unit test and a few more things.

C&C are welcomed.

### CODE REVIEW

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Community Code of Conduct
+
+Be nice to each others please.
```
Comment (courtneypattison) [inline .github/CODE_OF_CONDUCT.md:3]: other*

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-# Setting up a local development environment
+# Contribution guidelines
+
+We're thrilled that you'd like to contribute.  Your help is essential for making it better.
```
Comment (courtneypattison) [inline .github/CONTRIBUTING.md:3]: ...making Pattern Projector better.

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -49,7 +53,7 @@ on your local machine to check that building the app still works.
 
 Your PR will be picked up and reviewed before being merged into Pattern Projector proper.
 
-# Doing translations
+### Doing translations
 
 - Set up the development environment as described above.
```
Comment (courtneypattison) [inline .github/CONTRIBUTING.md:58]: The Doing translations section is out of date, sorry! The translations are done using Weblate. Here's the link: https://hosted.weblate.org/projects/pattern-projector/pattern-projector

**Comment by courtneypattison** [comment]
This looks great, thank you! Just some tiny wording changes.

I think it'd be simplest to leave architecture changes out of the CHANGELOG, especially since the CHANGELOG is linked to on the main site. What do you think?

Something that is also missing from the README_DEV is that we usually base pull requests off the beta branch. That branch is where the beta.patterprojector.com site comes from and acts as staging for the main site that I try to push to weekly.

Thank you again!

**Comment by irish1986** [comment]
Thanks I'll take a look at it later, sorry for my poor grammar I am French Canadian. I'll be looking into the translation later of course.

**Comment by courtneypattison** [comment]
I’m Canadian as well! The geese are back! I had to do some serious sidewalk poop dodging on my run this morning, lol.

**Comment by courtneypattison** [comment]
Merged with fixes in #237

**Pull request closed without merging**